openapi/paths/getters/getAliasesOnSale.json

```diff
@@ -0,0 +1,67 @@
+{
+  "get": {
+    "summary": "Get Aliases On Sale",
+    "description": "Get all aliases, which are available for sale.",
+    "tags": [
+      "alias"
+    ],
+    "parameters": [
+      {
+        "$ref": "../../parameters/firstIndex.json"
+      },
+      {
+        "$ref": "../../parameters/lastIndex.json"
+      }
+    ],
+    "responses": {
+      "200": {
+        "description": "The list of aliases this account owns.",
```

Comment:
Should this be "The list of aliases on sale"?

Reply:
good catch. you are totally right